vulcanai/plotters/__init__.py

```diff
@@ -12,6 +8,8 @@
     display_record,
     display_receptive_fields
 )
+import matplotlib
+matplotlib.use('agg')
```

Comment:
we want to make sure the `plt` import happens after the matplotlib backend get's set otherwise it won't change it after `plt` has been imported.

Reply:
ok. just reverted .